POC-OOBE/OOBE-Sandbox/PowerToys Settings Sandbox/Views/MainPage.xaml

```diff
@@ -17,73 +17,70 @@
-                        <TextBlock Text="3 of 3"  Margin="0,0,190,0" VerticalAlignment="Center"></TextBlock>
-                        <Button Background="White" Foreground="Blue" Content="Got it" Click="{x:Bind CloseTeachingTips}"></Button>
-                    </StackPanel>
-                </muxc:TeachingTip>
-                
-                <ToggleSwitch x:Name="RunStartupSwitch" Header="Run at startup" VerticalAlignment="Top"></ToggleSwitch>
+                <muxc:TeachingTip.IconSource>
+                    <muxc:SymbolIconSource Symbol="Help" />
+                </muxc:TeachingTip.IconSource>
+            </muxc:TeachingTip>
+
+            <ToggleSwitch x:Name="RunStartupSwitch" Header="Run at startup" VerticalAlignment="Top"></ToggleSwitch>
                 <ToggleSwitch Header="Download updates automatically" VerticalAlignment="Top"/>
 
                 <StackPanel Orientation="Horizontal">
                     <TextBlock x:Name="RunAsUserTitle" HorizontalAlignment="Left" Margin="0,24,0,12" Text="Running as user" TextWrapping="Wrap" VerticalAlignment="Top" Style="{StaticResource TitleTextBlockStyle}"/>
                     <FontIcon x:Name="RunAsUserTitleIcon"  FontFamily="Segoe MDL2 Assets" Glyph="&#xE946;" Margin="12,8,0,0" VerticalAlignment="Center" Foreground="Blue" PointerPressed="{x:Bind OpenRunAsUserTip}"/>
                 </StackPanel>
                 <muxc:TeachingTip x:Name="RunAsUserTip"
-                                    Target="{x:Bind RunAsUserTitleIcon}"
-                                    Background="Blue"
-                                    Foreground="White"
-                                    Title="Running as user"
-                                    PreferredPlacement="Right"
-                                    Subtitle="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Facere, nulla, ipsum? Molestiae quis aliquam magni harum non?"
-                                    >
-                    <StackPanel Orientation="Horizontal" Margin="0,12,0,0" HorizontalAlignment="Right">
-                        <TextBlock Text="2 of 3"  Margin="0,0,134,0" VerticalAlignment="Center"></TextBlock>
-                        <Button Background="Blue" Foreground="White" Margin="0,0,12,0" BorderBrush="White" Content="Dismiss" Click="{x:Bind CloseTeachingTips}"></Button>
-                        <Button Background="White" Foreground="Blue" Content="Next" Click="{x:Bind OpenFinalGeneralSettingsTip}"></Button>
-                    </StackPanel>
+                                        Target="{x:Bind RunAsUserTitleIcon}"
+                                        Foreground="Black"
+                                        Title="Running as user"
+                                        PreferredPlacement="Top"
+                                        Subtitle="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Facere, nulla, ipsum? Molestiae quis aliquam magni harum non?"
```

Comment:
Is content for the tooltips available yet?

Reply:
Still brainstorming and refining with user testing